index.bs

```diff
@@ -4508,10 +4508,8 @@ The following abstract operations operate on {{WritableStream}} instances at a h
     |stream|.[=WritableStream/[[controller]]=].[=WritableStreamDefaultController/[[signal]]=].
  1. Let |state| be |stream|.[=WritableStream/[[state]]=].
  1. If |state| is "`closed`" or "`errored`", return [=a promise resolved with=] undefined.
-
- <p class="note">We re-check the state because [=signaling abort=] runs author code and that might
- have changed the state.
-
+   <p class="note">We re-check the state because [=signaling abort=] runs author code and that might
+   have changed the state.
```

Comment:
Needs one more space for proper alignment. 😉
```suggestion
    <p class="note">We re-check the state because [=signaling abort=] runs author code and that might
    have changed the state.
```